### PR TITLE
Open Test Files

### PR DESCRIPTION
**Pull request by ramya-rao-a**
Implementing Feature Requests from #432 and #566

### CODE REVIEW

File: src/goGenerateTests.ts [hunk 1]
```diff
@@ -112,8 +152,29 @@ function generateTests(conf: Config): Thenable<boolean> {
 				if (err) {
 					return reject('Cannot generate test due to errors: ' + stderr);
 				}
-				let message = 'gotests: ' + conf.msg;
-				vscode.window.showInformationMessage(message);
+
+				let message = stdout;
+				let testsGenerated = false;
+				if (stdout.startsWith(generatedWord)) {
```
Comment (roblourens) [inline src/goGenerateTests.ts]: A comment about the output you expect and what you want to transform it to will make this easier to debug if it changes some day.

File: src/goGenerateTests.ts [hunk 2]
```diff
@@ -112,8 +152,29 @@ function generateTests(conf: Config): Thenable<boolean> {
...
+					}).map((element) => {
+						return element.substr(generatedWord.length);
+					});
+					message = `Generated ${lines.join(', ')}`;
+					testsGenerated = true;
+				}
+
+				if (testsGenerated) {
+					vscode.window.showInformationMessage(message, 'Open Generated Test for current file').then(selected => {
+						if (selected === 'Open Generated Test for current file') {
```
Comment (roblourens) [inline src/goGenerateTests.ts]: Extract this string? And i'm not sure 'Generated Test' should be capitalized.

Comment (ramya-rao-a) [inline src/goGenerateTests.ts]: will do

**Comment by kwiesmueller** [comment]
As I keep being annoyed by this feature in my daily work I was looking for a setting to disable it. As there is none I want to ask here.
When I work with tests in go, most of the time I got my editor in split. Left is code, right is test.
When then using the add test feature, my code gets closed and the test file (i already have open) gets opened there.
This is gets kinda annoying when generating a lot of tests.
My suggestion would be a setting for this to enable or disable it.
As an addition to that, I would also appreciate a setting to disable the "test created" modal, which blocks my view every time even if I don't care about.

What do you think?

**Pull request merged**